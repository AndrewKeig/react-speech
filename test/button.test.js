import React from 'react';
import Button from '../src/button';
import renderer from 'react-test-renderer';
import styles from '../src/style.js';

describe('Button', () => {
  describe('Button on load', () => {
    it('should render component', () => {
      const tree = renderer
        .create(<Button styles={styles.play} onClick={() => {}} />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
