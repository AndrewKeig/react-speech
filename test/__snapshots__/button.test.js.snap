// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Button on load should render component 1`] = `
<button
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={
    Object {
      "backgroundColor": "Gainsboro",
      "border": "solid 1px rgba(255,255,255,1)",
      "borderRadius": 6,
      "color": undefined,
      "cursor": "pointer",
      "height": "34",
      "outline": "none",
      "pointerEvents": "none",
      "width": "34",
    }
  }
  styles={
    Object {
      "button": Object {
        "backgroundColor": "Gainsboro",
        "border": "solid 1px rgba(255,255,255,1)",
        "borderRadius": 6,
        "cursor": "pointer",
        "height": "34",
        "outline": "none",
        "pointerEvents": "none",
        "width": "34",
      },
      "hover": Object {
        "backgroundColor": "GhostWhite",
      },
    }
  }
  type="button"
/>
`;
